requirements.txt

```diff
@@ -1,8 +1,4 @@
-tokenizers==0.10.3
-transformers==4.16.2
```

Comment:
fastapi 0.78.0 fails if headers are not the right content type (e.g application/json) had to add that explicitly to curl